concent_api/utils/helpers.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+import time
+
+import calendar
+
+from django.utils import timezone
+
+
+def get_current_timestamp() -> int:
+    """ Returns current timestamp as int. """
+    current_timestamp = int(timezone.now().timestamp())
+    assert calendar.timegm(time.gmtime()) - current_timestamp <= 1
+    return current_timestamp
+
+
+def parse_datetime_to_timestamp(date_time: datetime.datetime) -> int:
+    """ Returns timestamp as int from given datetime. """
+    return int(date_time.replace(tzinfo=timezone.utc).timestamp())
```

Comment:
I don't think this is correct for timezone-aware datetimes. It removes the existing `tzinfo` and puts UTC in its place without adjusting the values.

If you pass it a `datetime` with `tzinfo` indicating a timezone other than UTC it will simply ignore the timezone and convert to int as if it were in UTC.

This shows that we need tests for these helpers even though they may seem simple. Please add a test that passes various naive and tz-aware values (especially aware ones in different time zones) and compares resulting timestamps with our expectations. This will also come in handy if we decide to change the implementation later. We'll be sure that our assumptions are still satisfied.

Reply:
Right, I have tested it before and thought it is ok, but I must have messed up something with testing. I will change it and add tests for it.